unstract/connectors/src/unstract/connectors/databases/exceptions_helper.py

```diff
@@ -0,0 +1,8 @@
+class ExceptionHelper:
+    @staticmethod
+    def extract_byte_exception(e: Exception) -> tuple[str, str]:
```

Comment:
NIT: Please add a docstring on what we're trying to do - can this be extended for other generic exception handling cases? Also is there any chance of this exception handling code itself failing during `eval(error_message)`

Reply:
Added doc @chandrasekharan-zipstack . Let it be on the class itself, I checked the code, this class can be helper on other db connector later